.github/workflows/ci.yml

```diff
@@ -13,38 +11,36 @@ on:
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  # This workflow contains a single job called "build"
   build:
-    # The type of runner that the job will run on
-    runs-on: macos-latest
+    runs-on: macos-13
 
-    # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-
       # Checks-out the kiwix/apple repository
       - uses: actions/checkout@v4
         with:
           repository: kiwix/apple
           ref: main
           path: apple
     
-      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      # Checks-out the kiwix/kiwix-custom-apple repository
       - uses: actions/checkout@v4
         with:
           repository: kiwix/kiwix-apple-custom
```

Comment:
This is unecessaary as well, per default it takes current repository

Reply:
I would recommend something simple like:
```
      - name: Checkout code
        uses: actions/checkout@v4
```

Reply:
I have simplified it a bit. Had to keep the `path:` in, as I couldn't find another way to have 2 repositories checked out into the same workspace, they need to go under different sub folders.

Reply:
LGTM